controllers/slicegateway/types.go

```diff
@@ -60,5 +60,5 @@ type WorkerNetOpClientProvider interface {
 }
 
 type WorkerRecyclerClientProvider interface {
-	TriggerFSM(sliceGw *kubeslicev1beta1.SliceGateway, slice *kubeslicev1beta1.Slice, gatewayPod *corev1.Pod, controllerName, gwRecyclerName string, numberOfGwSvc int) (bool, error)
+	TriggerFSM(sliceGw *kubeslicev1beta1.SliceGateway, slice *kubeslicev1beta1.Slice, gatewayPod *corev1.Pod, controllerName, gwRecyclerName string, numberOfGwSvc int) error
```

Comment:
@YachikaRalhan let's add description here of `numberOfGwSvc` params and what this function does.
if possible add description about all the params